lib/src/sql/statements/define/analyzer.rs

```diff
@@ -11,23 +11,25 @@ use crate::sql::ending;
 use crate::sql::error::expected;
 use crate::sql::error::IResult;
 use crate::sql::filter::{filters, Filter};
-use crate::sql::ident::{ident, Ident};
+use crate::sql::ident::{ident, multi, Ident};
 use crate::sql::strand::{strand, Strand};
 use crate::sql::tokenizer::{tokenizers, Tokenizer};
 use crate::sql::value::Value;
 use derive::Store;
 use nom::branch::alt;
-use nom::bytes::complete::tag_no_case;
+use nom::bytes::complete::{tag, tag_no_case};
 use nom::combinator::cut;
 use nom::multi::many0;
 use revision::revisioned;
 use serde::{Deserialize, Serialize};
 use std::fmt::{self, Display};
 
 #[derive(Clone, Debug, Default, Eq, PartialEq, PartialOrd, Serialize, Deserialize, Store, Hash)]
-#[revisioned(revision = 1)]
+#[revisioned(revision = 2)]
 pub struct DefineAnalyzerStatement {
 	pub name: Ident,
+	#[revision(start = 2)]
```

Comment:
First time using "revision".  @tobiemh can you confirm that's the right way when we just add a field?

Reply:
Yeah that's good @emmanuel-keller ✅ .

@phughk yep, in this case the `Option<Ident>` implements `Default`, so if the field doesn't exist when deserialising, then it will be `None`.